### PR TITLE
Remove first paragraph - Closes #3156

### LINKED ISSUE #3156: Update README for the framework controller
### Expected behavior
Describe the changes to the framework controller made in Lisk Core 1.7.0.

### Which version(s) does this affect? (Environment, OS, etc...)
`1.6.0`

### PR DESCRIPTION
**Pull request by Tschakki**
Updated the controller Readme, removed unnecessary paragraph.

### Review checklist

* The PR resolves #3156 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

**Comment by MaciejBaj** [comment]
@Tschakki please open the PR against `release/2.0.0` branch.

File: framework/src/controller/README.md [hunk 1]
```diff
@@ -1,10 +1,5 @@
 # Controller
 
-## Description
-
-The controller is a parent process, that is responsible for managing every user interaction with each [component](../components/README.md) and [module](../modules/README.md) of the framework.
-E.g. restarting the node, launching a rebuild verification process, etc.
-
 The controller is responsible for initialization of the infrastructure-level components.
```
Comment (shuse2) [inline framework/src/controller/README.md]: I think this
> The controller is responsible for initialization of the infrastructure-level components.
is not true.


Controller does not initialize components.
Is this correct? @nazarhussain

Comment (nazarhussain) [reply framework/src/controller/README.md]: Yes that's partially correct. But here I guess its not referring those components, would be better to rephrase it. 

```suggestion
The controller is responsible for initialization the communication bus and any other dependencies required to load the modules. If any module is configured to load as a child process, then it is the controller's responsibility to do so.
```

File: framework/src/controller/README.md [hunk 2]
```diff
@@ -1,10 +1,5 @@
...
 The controller also initializes each module separately.
```
Comment (nazarhussain) [inline framework/src/controller/README.md]:
```suggestion
```

File: framework/src/controller/README.md [hunk 3]
```diff
@@ -1,10 +1,5 @@
...
 If any module is configured to load as a child process, then it is the controller's responsibility to do so.
```
Comment (nazarhussain) [inline framework/src/controller/README.md]:
```suggestion
```

**Pull request merged**